corehq/apps/sso/templates/sso/email/sso_deactivation_skip_notification.txt

```diff
@@ -0,0 +1,17 @@
+{% load i18n %}
+{% blocktrans %}
+Dear enterprise administrator,
+
+We have temporarily skipped automatic deactivate of SSO Web Users
+because we are receiving an empty list of users from your Microsoft Entra ID instance.
```

Comment:
```suggestion
We have temporarily skipped automatic deactivation of SSO Web Users
because we received an empty list of users from your Microsoft Entra ID instance.
```